micronota/bfillings/tests/data/NC_018498_partial_2.gbk

```diff
@@ -0,0 +1,632 @@
+DEFINITION  Mycoplasma genitalium M2288 chromosome, complete genome.
+ACCESSION   NC_018498
+VERSION     NC_018498.1  GI:402552294
+DBLINK      Project: 173372
+KEYWORDS    .
+SOURCE      Mycoplasma genitalium M2288
+  ORGANISM  Mycoplasma genitalium M2288
+            Bacteria; Tenericutes; Mollicutes; Mycoplasmataceae; Mycoplasma.
+REFERENCE   1  (bases 1 to 579558)
+  AUTHORS   McGowin,C.L., Ma,L., Jensen,J.S., Mancuso,M.M., Hamasuna,R., Adegboye,D. and Martin,D.H.
+  TITLE     Draft genome sequences of four axenic Mycoplasma genitalium strains isolated from Denmark, Japan, and Australia
+  JOURNAL   J. Bacteriol. 194 (21), 6010-6011 (2012)
+  PUBMED    23045512
+REFERENCE   2  (bases 1 to 579558)
+  TITLE     Direct Submission
+  JOURNAL   Submitted (21-AUG-2012) National Center for Biotechnology Information, NIH, Bethesda, MD 20894, USA
+  CONSRTM   NCBI Genome Project
+REFERENCE   3  (bases 1 to 579558)
+  AUTHORS   McGowin,C.L., Ma,L., Jensen,J.S., Mancuso,M.M., Hamasuna,R., Adegboye,D. and Martin,D.H.
+  TITLE     Direct Submission
+  JOURNAL   Submitted (14-AUG-2012) Department of Medicine; Section of Infectious Diseases, Louisiana State University Health Sciences Center, 533 Bolivar Street - CSRB 701, New Orleans, LA 70112, USA
+COMMENT     PROVISIONAL REFSEQ: This record has not yet been subject to final NCBI review. The reference sequence is identical to CP003773. Annotation was added by the NCBI Prokaryotic Genomes Automatic Annotation Pipeline Group. Information about the Pipeline can be found here: http://www.ncbi.nlm.nih.gov/genomes/static/Pipeline.html. Please be aware that the annotation is done automatically with little or no manual curation. COMPLETENESS: full length.
+FEATURES             Location/Qualifiers
+       source        1..22350
+                     /db_xref="taxon:662947"
+                     /mol_type="genomic DNA"
+                     /organism="Mycoplasma genitalium M2288"
+                     /strain="M2288"
+         gene        686..1828
+                     /db_xref="GeneID:13492420"
+                     /locus_tag="CM5_00005"
+          CDS        686..1828
+                     /codon_start=1
+                     /db_xref="GI:402552295"
+                     /db_xref="GeneID:13492420"
+                     /locus_tag="CM5_00005"
+                     /note="COG0592 DNA polymerase sliding clamp subunit (PCNA homolog)"
+                     /product="DNA polymerase III subunit beta"
+                     /protein_id="YP_006601012.1"
+                     /transl_table=4
+                     /translation="MKILINKSELNKILKKMNNVIISNNKIKPHHSYFLIEAKEKEIN FYANNEYFSVKCNLNKNIDILEQGSLIVKGKIFNDLINGIKEEIITIQEKDQTLLVKT KKTSINLNTINVNEFPRIRFNEKNDLSEFNQFKINYSLLVKGIKKIFHSVSNNREISS KFNGVNFNGSNGKEIFLEASDTYKLSVFEIKQETEPFDFILESNLLSFINSFNPEEDK SIVFYYRKDNKDSFSTEMLISMDNFMISYTSVNEKFPEVNYFFEFEPETKIVVQKNEL KDALQRIQTLAQNERTFLCDMQINSSELKIRATVNNIGNSLEEISCLKFEGYKLNISF NPSSLLDHIESFESNEINFDFQGNSKYFLITSKSEPELKQILVPSR"
```

Comment:
it doesn't matter for the format.

On Tue, Oct 13, 2015 at 4:29 PM, mortonjt <notifications@github.com> wrote:

> In micronota/bfillings/tests/data/NC_018498_partial_2.gbk
> <https://github.com/biocore/micronota/pull/4#discussion_r41939960>:
>
> > +                     /mol_type="genomic DNA"
> > +                     /organism="Mycoplasma genitalium M2288"
> > +                     /strain="M2288"
> > +         gene        686..1828
> > +                     /db_xref="GeneID:13492420"
> > +                     /locus_tag="CM5_00005"
> > +          CDS        686..1828
> > +                     /codon_start=1
> > +                     /db_xref="GI:402552295"
> > +                     /db_xref="GeneID:13492420"
> > +                     /locus_tag="CM5_00005"
> > +                     /note="COG0592 DNA polymerase sliding clamp subunit (PCNA homolog)"
> > +                     /product="DNA polymerase III subunit beta"
> > +                     /protein_id="YP_006601012.1"
> > +                     /transl_table=4
> > +                     /translation="MKILINKSELNKILKKMNNVIISNNKIKPHHSYFLIEAKEKEIN FYANNEYFSVKCNLNKNIDILEQGSLIVKGKIFNDLINGIKEEIITIQEKDQTLLVKT KKTSINLNTINVNEFPRIRFNEKNDLSEFNQFKINYSLLVKGIKKIFHSVSNNREISS KFNGVNFNGSNGKEIFLEASDTYKLSVFEIKQETEPFDFILESNLLSFINSFNPEEDK SIVFYYRKDNKDSFSTEMLISMDNFMISYTSVNEKFPEVNYFFEFEPETKIVVQKNEL KDALQRIQTLAQNERTFLCDMQINSSELKIRATVNNIGNSLEEISCLKFEGYKLNISF NPSSLLDHIESFESNEINFDFQGNSKYFLITSKSEPELKQILVPSR"
>
> Should these translation have spaces?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/micronota/pull/4/files#r41939960>.
>